src/spec.c

```diff
@@ -992,7 +992,7 @@ char *IndexSpec_GetRandomTerm(IndexSpec *sp, size_t sampleSize) {
 
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-void IndexSpec_FreeInternals(IndexSpec *spec) {
+void IndexSpec_FreeGlobals(IndexSpec *spec) {
```

Comment:
Can you explain this function and how is it different from `IndexSpec_FreeInternals`. When each of them should be used?